pihole_adlist_tool

```diff
@@ -438,12 +441,12 @@ if [ "$ENABLE_ALL_ADLISTS_FOR_ANALYSIS" -eq 1 ]; then
 
     # check if a mismatch between enabled adlists and data in gravity exists, offer to run gravity
     # https://stackoverflow.com/a/28161520
-    adlist_enabled_in_gravity=(`sqlite $GRAVITY "select distinct adlist_id from gravity;"`)
-    if [ -n "$(echo ${adlist_conf_old_enabled[@]} ${adlist_enabled_in_gravity[@]} | tr ' ' '\n' |sort |uniq -u)" ]; then
+    mapfile -t adlist_enabled_in_gravity < <(sqlite "$GRAVITY" "select distinct adlist_id from gravity;")
```

Comment:
```suggestion
    mapfile -t adlist_enabled_in_gravity < <(sqlite $GRAVITY "select distinct adlist_id from gravity;")
```

That's the reason `adlist_enabled_in_gravity` is zero

Reply:
This seems not to be the problem nor the solution. I tried with old version right from beginning of this PR (`commit 375235746a4590031be96bf4d348bce24b00275d (HEAD, tag: 2.6.0)`). Its output produces also **no** "mismatch between your enabled adlists and the data found in the gravity database". And I can't reproduce any mismatch again 🤷🏻‍♂️

Reply:
Ok, so currently there is no output difference between old and new version?

Could you address the suggestion above please?